app/views/unattended/provisioning_templates/host_init_config/host_init_config_default.erb

```diff
@@ -54,6 +54,11 @@ trap 'exit_and_cancel_build' ERR
 
 <% end -%>
 
+<% if plugin_present?('katello') && @host.operatingsystem.name == 'Redhat' -%>
```

Comment:
```
irb(main):003:0> Host.first.operatingsystem.name
=> "RedHat"
```

Shouldn't there be capital 'H'?

Reply:
You are right, `name` is `RedHat` but `family` is `Redhat`, easy to make mistake. Fixed

Reply:
Fixed again (sick), now using `family` instead of `name`, as we want to do it for RHEL, CentOS, Rocky and so on ...